docs/networking/networking.md

```diff
@@ -3,7 +3,7 @@ title: "Networking"
 weight: 35
 ---
 
-This page explains how CoreDNS, the Traefik Ingress controller, and Klipper service load balancer work within K3s.
+This page explains how CoreDNS, the Traefik Ingress controller, Network Policy Controller and Klipper service load balancer work within K3s.
```

Comment:
Oxford comma. Also, I think we're not calling stuff Klipper so much any more.
```suggestion
This page explains how CoreDNS, Traefik Ingress controller, Network Policy controller, and ServiceLB load balancer controller work within K3s.
```

Reply:
thanks!